utils/apiValidators.ts

```diff
@@ -0,0 +1,43 @@
+import { ApiError } from './types'
+import {
+  validateAttribute,
+  validateCategory,
+  validateItemDefinition,
+  validateObjectId,
+  validateUser,
+  ValidationResult,
+} from './validators'
+
+const BAD_REQUEST_BODY_PREFIX = 'Bad Request Body: '
+
+export function apiObjectIdValidation(id: any) {
+  if (!validateObjectId(id)) {
+    throw new ApiError(400, 'Invalid ObjectId Format')
+  }
+}
+
+export function apiAttributeValidation(attribute: any) {
```

Comment:
rather than accepting anything here and in all other `api{type}Validation` funcs, signify you are accepting any object by using the type `Record<string, unknown>`

Reply:
fixed